experimenter/tests/integration/nimbus/test_experimenter_ui.py

```diff
@@ -106,3 +107,60 @@ def test_every_form_page_can_be_resaved(
     audience = metrics.save_and_continue()
     summary = audience.save_and_continue()
     assert summary.experiment_slug is not None
+
+
```

Comment:
I think I might know why the tests are failing, you are missing a mark ```@pytest.mark.nimbus_ui```. Add this mark to the tests and try again